CODE_OF_CONDUCT.md

```diff
@@ -9,6 +9,6 @@ Specifically:
 - Be kind. Be courteous. Be welcoming.
 - Listen. Consider and acknowledge people’s points before responding.
 
-If any member of the community violates this code of conduct, the maintainers of the Pharo project may take action, removing issues, comments, and PRs or blocking accounts as deemed appropriate. We will not tolerate harassment from anyone in the Pharo community, even outside of Pharo’s public communication channels.
+If any member of the community violates this code of conduct, the maintainers of the Pharo project may take action, removing issues, comments, and PRs or blocking accounts as deemed appropriate. We will not tolerate harassment from anyone in the Pharo community.
 
-If you are subject to or witness unacceptable behavior, or have any other concerns, please [email us](board@pharo.org).
\ No newline at end of file
+If you are subject to or witness unacceptable behavior, or have any other concerns, please [email us](board@pharo.org).
```

Comment:
[To imply that there is an openness to hear both sides and that there is some due process, I invite consideration of the following:]

If you are subject to *, accused of,* or witness unacceptable behavior, or have any other concerns, please [email us](board@pharo.org).

Reply:
I think is reasonable. 
PR please :)